arch/arm64/configs/eve_defconfig

```diff
@@ -731,6 +735,9 @@ CONFIG_PINCTRL_IMX8QM=y
 CONFIG_PINCTRL_IMX8QXP=y
 CONFIG_PINCTRL_IMX8DXL=y
 CONFIG_PINCTRL_SPRD_SC9860=y
+CONFIG_PINCTRL_BCM2712=y
```

Comment:
It's really needed to built-in all these configs? Our kernel it's an ARM64 generic kernel that can run on other devices as well, so if there is something that can be built as a module, would be better to let as a module...